module2/homework/insensitivePalindrom/is_palindrome.hpp

```diff
@@ -0,0 +1,12 @@
+#include <algorithm>
+#include <cctype>
+#include <string>
+
+bool is_palindrome(std::string text)
+{
+    text.erase(std::remove_if(text.begin(), text.end(), [](unsigned char x) { return !std::isalpha(x); }), text.end());
+    std::transform(text.begin(), text.end(), text.begin(), [](unsigned char c) { return std::tolower(c); });
+    std::string text_cp = text;
+    reverse(text_cp.begin(), text_cp.end());
+    return text == text_cp;
```

Comment:
it is easier way to do this (without creating whole new string which is reversed)

Reply:
hm, with reverse iterator and std::equal ?

Reply:
Yes, exactly

Reply:
+1